.github/workflows/ci_ubuntu.yml

```
@@ -80,8 +80,8 @@ jobs:
       - name: Test in Ubuntu
         run: |
           cd ./build
-          make coverage
-          make pythontest
+          make coverage -j
+          make pythontest -j
```

concurrency rate should be limited to number of cpu cores.

```suggestion
          make coverage -j $(nproc)
          make pythontest -j $(nproc)
```